Math.enum.h

```diff
@@ -19,20 +19,22 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-/**
- * @file
- * Includes Math's enums.
- */
+// @file
+// Includes Math's enums.
```

Comment:
This one was actually fine, change it to what it was. It's doc block for the file description.